From e4579214961f3954101254857b995fe6c8ae7cd1 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 19 Jul 2022 14:49:56 +0800
Subject: [PATCH 39/62] Revert "drm/msm/hdmi: check return value after calling
 platform_get_resource_byname()"

This reverts commit d9cb951d11a4ace4de5c50b1178ad211de17079e.
---
 drivers/gpu/drm/msm/hdmi/hdmi.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/msm/hdmi/hdmi.c b/drivers/gpu/drm/msm/hdmi/hdmi.c
index 2758b51aa4e0..94f948ef279d 100644
--- a/drivers/gpu/drm/msm/hdmi/hdmi.c
+++ b/drivers/gpu/drm/msm/hdmi/hdmi.c
@@ -142,10 +142,6 @@ static struct hdmi *msm_hdmi_init(struct platform_device *pdev)
 	/* HDCP needs physical address of hdmi register */
 	res = platform_get_resource_byname(pdev, IORESOURCE_MEM,
 		config->mmio_name);
-	if (!res) {
-		ret = -EINVAL;
-		goto fail;
-	}
 	hdmi->mmio_phy_addr = res->start;
 
 	hdmi->qfprom_mmio = msm_ioremap(pdev,
-- 
2.27.0

